x-pack/plugin/security/build.gradle

```diff
@@ -73,7 +73,7 @@ dependencies {
   api "com.nimbusds:lang-tag:1.4.4"
   api "com.sun.mail:jakarta.mail:1.6.3"
   api "net.jcip:jcip-annotations:1.0"
-  api "net.minidev:json-smart:2.4.2"
```

Comment:
My only concern here is whether we can update json-smart on its own given that this is a dependency of oauth2-oidc-sdk and the version 9.3.1 we are using seems to be compiled against/tested with different versions of json-smart (https://bitbucket.org/connect2id/oauth-2.0-sdk-with-openid-connect-extensions/src/9.3.1/pom.xml)

Reply:
Very good point, replied on the internal issue.